### PR TITLE
Adding Collapsible to materialize-css/index.d.ts

### PR DESCRIPTION
**Pull request by MonizDave**
Please fill in this template.

- [x] Use a meaningful title for the pull request. Include the name of the package modified.
- [x] Test the change in your own code. (Compile and run.)
- [x] Add or edit tests to reflect the change. (Run with `npm test`.)
- [x] Follow the advice from the [readme](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#make-a-pull-request).
- [x] Avoid [common mistakes](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#common-mistakes).
- [x] Run `npm run lint package-name` (or `tsc` if no `tslint.json` is present).

Select one of these and delete the others:

If changing an existing definition:
Just adding Collapsible to materialize-css/index.d.ts
added corresponding tests

### CODE REVIEW

File: types/materialize-css/index.d.ts [hunk 1]
```diff
@@ -77,6 +77,71 @@ declare global {
...
+            inDuration: number;
+
+            /**
+             * Transition out duration in milliseconds
+             * @default 300
+             */
+            outDuration: number;
+
+            /**
+             * Callback function called before collapsible is opene
```
Comment (huww98) [inline types/materialize-css/index.d.ts]: A `d` is missing at the end of this line.

**Review by huww98 — CHANGES_REQUESTED** [review]
Thanks. Just a small typo.

Can you please also fix the `Definitions by:` header? It should be one person per line. Also you may add yourself to this list.

**Comment by broccoliarchy** [comment]
This is included in #25448

**Comment by MonizDave** [comment]
@broccoliarchy any reason you included my changes but left my name off of the definition by section?

**Comment by broccoliarchy** [comment]
@MonizDave I saw this PR after I completed the other, so that PR does not include code from this PR - they just overlap. However, I'd be happy to credit you there if you like.

**Comment by MonizDave** [comment]
@broccoliarchy if you don't mind

**Pull request closed without merging**